aea/crypto/base.py

```diff
@@ -152,7 +152,48 @@ def dump(self, fp: BinaryIO) -> None:
         """
 
 
-class LedgerApi(ABC):
+class Helper(ABC):
```

Comment:
The helper class does not need network access. It can be exposed on the skill context eventually. It also can serve as a mixin to the ledger api.